CondFormats/L1TObjects/interface/L1TMuonBarrelKalmanParams.h

```diff
@@ -0,0 +1,96 @@
+///
+/// \class L1TMuonBarrelKalmanParams
+///
+/// Description: Placeholder for Kalman BMTF parameters
+///
+///
+/// \author: Panos Katsoulis
+///
+
+#ifndef L1TBMTFKalmanParams_h
+#define L1TBMTFKalmanParams_h
+
+#include <memory>
+#include <iostream>
+#include <vector>
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+#include "CondFormats/L1TObjects/interface/LUT.h"
+
+class L1TMuonBarrelKalmanParams {
+public:
+  L1TMuonBarrelKalmanParams();
+  ~L1TMuonBarrelKalmanParams() {}
+
+  enum { Version = 1 };
+  enum { CONFIG = 0, NUM_BMTF_PARAM_NODES = 2 };
+
+  class Node {
+  public:
+    std::string type_;
+    std::string kalmanLUTsPath_;
+    unsigned fwVersion_;
+    l1t::LUT LUT_;
+    /* std::vector<double> dparams_; */
```

Comment:
Is all this commented out code still needed in the file?
Either add a comment line which explains why it must stay there, or better clean up and remove (here and below)

Reply:
Hi, this PR is the baseline for having CondDB params in the BMTF Kalman filter Emu.
This will be finished by other people in the next months (less experienced)
These commented out lines are the required parameters for the algo in order to make in the future the implementation faster.
I think it's gonna be useful, but don't know if there is a policy to not include at all code to be used in the future.
Please inform me if it's not fine to keep it for now